marklogic-data-hub-central/ui/src/pages/Browse.tsx

```diff
@@ -352,6 +353,7 @@ const Browse: React.FC<Props> = ({location}) => {
     }
     return () => {
       setGraphSearchData({});
+      setSavedNode(undefined);
```

Comment:
Can we move this line to line 173? It makes more sense to only apply this when we are in graph view and we have new graphSearchData.
In this line it is being called eveytime there is a change in the dependencies regardless if we are in graph or not.

Reply:
Placing it after the graphSearch endpoint might result in the side panel disappearing after few seconds if the dataset is huge. Added it in line 354 instead, to ensure it will be called only when needed.